bindings/node/README.md

```diff
@@ -242,10 +266,14 @@ Configuration options for a automatic client encryption.
 | [mongocryptdURI] | <code>string</code> |  | A local process the driver communicates with to determine how to encrypt values in a command. Defaults to "mongodb://%2Fvar%2Fmongocryptd.sock" if domain sockets are available or "mongodb://localhost:27020" otherwise |
 | [mongocryptdBypassSpawn] | <code>boolean</code> | <code>false</code> | If true, autoEncryption will not attempt to spawn a mongocryptd before connecting |
 | [mongocryptdSpawnPath] | <code>string</code> |  | The path to the mongocryptd executable on the system |
-| [mongocryptdSpawnArgs] | <code>Array.&lt;string&gt;</code> |  | Command line arguments to use when auto-spawning a mongocryptd |
+| [mongocryptdSpawnArgs] | <code>string[]</code> |  | Command line arguments to use when auto-spawning a mongocryptd |
+| [cryptSharedLibPath]   | <code>string</code> |  | Full path to a MongoDB Crypt shared library on the system. If specified, autoEncryption will not attempt to spawn a mongocryptd, but makes use of the shared library file specified. Note that the path must point to the shared libary file itself, not the folder which contains it. \*|
+| [cryptSharedLibRequired]  | <code>boolean</code> | | If true, never use mongocryptd and fail when the MongoDB Crypt shared libary cannot be loaded. Defaults to true if [cryptSharedLibPath] is specified and false otherwise. \*|
```

Comment:
https://github.com/mongodb/libmongocrypt/blob/NODE-5261/update_readme_to_mention_cryptSharedLibPath/bindings/node/lib/autoEncrypter.js#L25

This should be added the JSdoc comments, `npm run docs` currently wipes out these changes